tangos/testing/__init__.py

```diff
@@ -160,12 +162,21 @@ def init_blank_db_for_testing(**init_kwargs):
 
     testing_db_name = init_kwargs.pop("testing_db_name", caller_fname)
 
-    db_name = "test_dbs/%s.db"%testing_db_name
-    try:
-
-        os.remove(db_name)
-    except OSError:
-        pass
-
-    core.init_db("sqlite:///"+db_name,**init_kwargs)
+    if testing_db_backend == "sqlite":
+        db_name = f"test_dbs/%s.db"%testing_db_name
+        try:
+            os.remove(db_name)
+        except OSError:
+            pass
 
+        core.init_db(f"sqlite:///{db_name}", **init_kwargs)
+    else:
+        db_url = f"{testing_db_backend}://{testing_db_user}:{testing_db_password}@localhost"
+        engine = create_engine(db_url)
+        with engine.connect() as conn:
+            conn.execute("COMMIT")
+            # Do not substitute user-supplied database names here.
```

Comment:
I'm not sure I understand this comment, what's it saying?

Reply:
I think this is a leftover from a previous version, I'm going to delete it.